doc/exits-and-hotspots.md

```diff
@@ -0,0 +1,58 @@
+# Hotspots and exits
+
+Hotspots in Escoria are thought of as "things a character can walk on", either
+like opening a door in Prince of Persia by stepping on a plate or a more abstract
+"To street" exit hotspot.
+
+Inert items, things that are "hotspots for the mouse" but not to be walked on, should
+be implemented using `item.gd`.
+
+Although hotspots and exits can be attached to any type of `Node`,
+not every case works properly.
+
+The working cases are described here.
+
+Exit is extended from hotspot. To make gameplay quicker and "more modern", hotspot
+implements double-clicking, which therefore works in exit as well.
+
+## Concerning the player
+
+Your player character must be a `KinematicBody2D` in order to work with hotspots
+and exits.
+
+If you refer to an old documentation that claims otherwise, and don't make it a
+`KinematicBody2D`, Godot will complain. So before you start, remember this.
+
+Also bear in mind that your player will need a `CollisionShape2D` or such to
+collide with the hotspots and exits.
+
+## Hotspots
+
+Hotspots are `Area2D`, with the required `CollisionPolygon2D` child and optionally also
+a `Sprite`) child. You may name them freely.
+
+The `hotspot.gd` script must be attached to the `Area2D`.
+
+The hotspots react only on the player character.
```

Comment:
I was making another point than you. I meant how an NPC will not (currently) trigger an event there.

I expanded on it now, though, to include what you said. I preferred the word "within" because included sounds a bit different.

Force-pushed and while at it, reworded the commit slightly for better `git blame`.